From mboxrd@z Thu Jan  1 00:00:00 1970
Return-Path: <linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
Received: from bombadil.infradead.org (bombadil.infradead.org [198.137.202.133])
	(using TLSv1.2 with cipher ECDHE-RSA-AES256-GCM-SHA384 (256/256 bits))
	(No client certificate requested)
	by smtp.lore.kernel.org (Postfix) with ESMTPS id 1CCE7D30CC2
	for <linux-rockchip@archiver.kernel.org>; Tue, 13 Jan 2026 20:00:03 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; q=dns/txt; c=relaxed/relaxed;
	d=lists.infradead.org; s=bombadil.20210309; h=Sender:
	Content-Transfer-Encoding:Content-Type:List-Subscribe:List-Help:List-Post:
	List-Archive:List-Unsubscribe:List-Id:MIME-Version:References:In-Reply-To:
	Message-ID:Date:Subject:Cc:To:From:Reply-To:Content-ID:Content-Description:
	Resent-Date:Resent-From:Resent-Sender:Resent-To:Resent-Cc:Resent-Message-ID:
	List-Owner; bh=7sKwZhuXyDvX7JyAL+3IAJ8f05NaAKn48bMXFFLQSBQ=; b=mKYISh+Fu7TAk1
	8aam6NoJALFrLi4Zh5j3ldnN66yaUB9JAaUitu1Mi6m4YcYD1Aad+SkopmK1Pz0WhIhzrKTeQeUBi
	dvPwUZ6SJAhmWgy04WgvCsxtGxrnJFzYOICWh9noMDIU/HNl92nZcKzzSDpBDqU8rks98F3lt2a1X
	88OlyeeFeXv+x6YYZwj74FAkzrELijOUTeA6+AbXWe0WAQ8jV12ImjWFn6+IhABOIB7d8XeumDBxP
	Ucn8mpEmcDVjJaGvp3m8NGeLqyR9ja6R6fiX0KuhWVx8Y44Qh9EkvR2Hf26KroJnDAL0o/9S9+DfS
	5Gvcpv55wSSWWRpFqptA==;
Received: from localhost ([::1] helo=bombadil.infradead.org)
	by bombadil.infradead.org with esmtp (Exim 4.98.2 #2 (Red Hat Linux))
	id 1vfkYW-00000007ebQ-1oN5;
	Tue, 13 Jan 2026 19:59:56 +0000
Received: from mail-ot1-x32e.google.com ([2607:f8b0:4864:20::32e])
	by bombadil.infradead.org with esmtps (Exim 4.98.2 #2 (Red Hat Linux))
	id 1vfkYT-00000007eZH-2kM8
	for linux-rockchip@lists.infradead.org;
	Tue, 13 Jan 2026 19:59:54 +0000
Received: by mail-ot1-x32e.google.com with SMTP id 46e09a7af769-7c6d13986f8so162695a34.0
        for <linux-rockchip@lists.infradead.org>; Tue, 13 Jan 2026 11:59:53 -0800 (PST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=gmail.com; s=20230601; t=1768334392; x=1768939192; darn=lists.infradead.org;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:cc:to:from:from:to:cc:subject:date
         :message-id:reply-to;
        bh=M8NK3Ykk7hfqolFaWL1u6Km9KABPeFUAOu8kCyf074c=;
        b=NdULPPO6gJBpYLlAX915E/Z+aIjIeza4E9L8DvKI1zZoRMQRjyq0aCTpIg/AcGYAr5
         kugqBJaIrQ0XmMQ0QzLdNd5/xGTX4unGusoQY+xsoQqF4kJhctzuhJRffEnXWH9kODfE
         jj2iRRRIVNBZas0wpR/YmWbawcvFYTzz7+HZb7ifs3vlqWuwCUpIEwkOWOJHbVseZdEE
         tZUnJd9b+5Kxp2nBWJG9imS7ui7OVfqrV7OIErxvlQ8EYJe+OSqz4Vj89gINXTtRzTvd
         bC0JD6ydppTQZHHFZI6WIyXNP3yAxwOT5xIReBvE8mzmp+Pgfv3M3sgPvLBxVv929c1j
         QzlA==
X-Google-DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed;
        d=1e100.net; s=20230601; t=1768334392; x=1768939192;
        h=content-transfer-encoding:mime-version:references:in-reply-to
         :message-id:date:subject:cc:to:from:x-gm-gg:x-gm-message-state:from
         :to:cc:subject:date:message-id:reply-to;
        bh=M8NK3Ykk7hfqolFaWL1u6Km9KABPeFUAOu8kCyf074c=;
        b=KDoXzBSZCpyJ17VwN91QvlmKtPfLFXfVivUS98GGJz/j6cRU9AkymADoxL+NHCfi8K
         Ub88D8XnLRqAL+5ZCSdvFqqTJZsxMEg2kXIT3Op90Y0+BJm9vN6qp+VOu008uV2lJbOa
         FvcmIPzINNZR/CDDEWZ+syGtF60zcf+HDdjkODPBtr8huzjwrKW5w0IecjgdqkKwT4Yl
         R6fBLwRb1EXmwKS6cIRYJwNft8O0zbXKBQRwbXr24V3rUTMITvU7LEvsJyM/BMbvZezt
         nZ6WKqRG269Sai+g+PhMwAi5QTStgM3eZC49+t2Im368v+880pOzEO/GSgh5tD/Yc92R
         BBZA==
X-Gm-Message-State: AOJu0YwtEaBY0Bnp6rwuvmcsikVJSnkpV/clP6xSDpi+8RWiq2In+Lbs
	nzo0PZPmmkriX7O7lm2iWJGHD9T3WIoBvsIHNm/579TdtC4TIYJJviO76D0yHQ==
X-Gm-Gg: AY/fxX7ozfJAFwRNbLRNQypRLPlYGMhDTk1LAN0VrR8zfP2/OkWp2DYuJV3+09H69FC
	QAUOXTasWmxWJvybZ60YH3iJwDfl+TMDn1w71RdiHrjNSy6lzacqB/DUgmuc7Ij2ykSvqLLh2Mt
	jkF9GAtbG+hyN5/f7N5ucaFjlXx6TFuZ8ZDff/Ys1KKmrgD45Cd8pCnnxpUtCVhKZ0VOKp1i82i
	XO1cN37FoSv1A82iVfNRlr/oJew70wgM03fnOiOrDYo1J3IP5T6KmghPAMmck42Qt8TtIW/3oiS
	8lmDWQhQ0GgSz4sKwDzTOORkzIb21RqYEdSd5xx9V9nyHxvnxEbDhiFEUy2LaDupRjtyB2vf0t9
	R9vM5Dd56eRfKOfEpQYhwUio1+BIPNkFwfyCx7U0pJivd2dUPAXYxOHPZjYdqe7y4h1hwlPguO/
	b0haLpjUk3
X-Received: by 2002:a05:6830:2b07:b0:7c7:e3b:4860 with SMTP id 46e09a7af769-7cfc91608camr140237a34.10.1768334392359;
        Tue, 13 Jan 2026 11:59:52 -0800 (PST)
Received: from localhost.localdomain ([2600:1700:fb0:1bc0::54])
        by smtp.gmail.com with ESMTPSA id 46e09a7af769-7ce47801d4dsm16221119a34.5.2026.01.13.11.59.51
        (version=TLS1_3 cipher=TLS_AES_256_GCM_SHA384 bits=256/256);
        Tue, 13 Jan 2026 11:59:51 -0800 (PST)
From: Chris Morgan <macroalpha82@gmail.com>
To: linux-rockchip@lists.infradead.org
Cc: dri-devel@lists.freedesktop.org,
	devicetree@vger.kernel.org,
	dmitry.torokhov@gmail.com,
	simona@ffwll.ch,
	airlied@gmail.com,
	tzimmermann@suse.de,
	mripard@kernel.org,
	maarten.lankhorst@linux.intel.com,
	jesszhan0024@gmail.com,
	neil.armstrong@linaro.org,
	jagan@edgeble.ai,
	heiko@sntech.de,
	conor+dt@kernel.org,
	krzk+dt@kernel.org,
	robh@kernel.org,
	aweinzerl13@yahoo.com,
	Chris Morgan <macromorgan@hotmail.com>
Subject: [PATCH 5/6] dt-bindings: input: touchscreen: goodix: Add "panel" property
Date: Tue, 13 Jan 2026 13:57:20 -0600
Message-ID: <20260113195721.151205-6-macroalpha82@gmail.com>
X-Mailer: git-send-email 2.43.0
In-Reply-To: <20260113195721.151205-1-macroalpha82@gmail.com>
References: <20260113195721.151205-1-macroalpha82@gmail.com>
MIME-Version: 1.0
X-CRM114-Version: 20100106-BlameMichelson ( TRE 0.8.0 (BSD) ) MR-646709E3 
X-CRM114-CacheID: sfid-20260113_115953_695253_7DCA26CC 
X-CRM114-Status: UNSURE (   9.63  )
X-CRM114-Notice: Please train this message.
X-BeenThere: linux-rockchip@lists.infradead.org
X-Mailman-Version: 2.1.34
Precedence: list
List-Id: Upstream kernel work for Rockchip platforms <linux-rockchip.lists.infradead.org>
List-Unsubscribe: <http://lists.infradead.org/mailman/options/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=unsubscribe>
List-Archive: <http://lists.infradead.org/pipermail/linux-rockchip/>
List-Post: <mailto:linux-rockchip@lists.infradead.org>
List-Help: <mailto:linux-rockchip-request@lists.infradead.org?subject=help>
List-Subscribe: <http://lists.infradead.org/mailman/listinfo/linux-rockchip>,
 <mailto:linux-rockchip-request@lists.infradead.org?subject=subscribe>
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit
Sender: "Linux-rockchip" <linux-rockchip-bounces@lists.infradead.org>
Errors-To: linux-rockchip-bounces+linux-rockchip=archiver.kernel.org@lists.infradead.org

From: Chris Morgan <macromorgan@hotmail.com>

Add a "panel" property to define a relationship between a touchscreen
and an associated panel when more than one of each exist in a device.

Signed-off-by: Chris Morgan <macromorgan@hotmail.com>
---
 Documentation/devicetree/bindings/input/touchscreen/goodix.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml b/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml
index a96137c6f063..a26a54d63a1c 100644
--- a/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml
+++ b/Documentation/devicetree/bindings/input/touchscreen/goodix.yaml
@@ -42,6 +42,8 @@ properties:
       address, thus it can be driven by the host during the reset sequence.
     maxItems: 1
 
+  panel: true
+
   reset-gpios:
     maxItems: 1
 
-- 
2.43.0


_______________________________________________
Linux-rockchip mailing list
Linux-rockchip@lists.infradead.org
http://lists.infradead.org/mailman/listinfo/linux-rockchip

